packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -275,7 +275,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
         // URIs use slashes as delimiters. Handles use URIs.
         // Thus having slashes in types almost guarantees trouble down the road!
-        assert(!this.id.includes("/"), 0x13a /* `Data store ID contains slash: ${id}` */);
+        if (this.id.includes("/")) {
```

Comment:
We should probably have different validations for `Local` and 'Remote' contexts.
1. Local context is created when a user creates a data store via the create APIs so that should results in a `UsageError`.
2. Remote context is created by the runtime for existing data stores when it processes attach op or loads from summary. It should never have slashes because you cannot create a data store with slashes (1/ above). So, we should leave this as assert.